### PR TITLE
adding i5_i7_index

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by wasade** [comment]
Should there be any data access changes?

File: amgut/db/patches/0041.sql [hunk 1]
```diff
@@ -15,13 +15,21 @@ CREATE TABLE pm.shotgun_adapter_aliquot (
...
+	name                 varchar  NOT NULL,
+	notes                varchar  ,
+	limit_freeze_thaw_cycles bigint  NOT NULL,
+	CONSTRAINT pk_index_aliquot PRIMARY KEY ( shotgun_index_aliquot_id ),
+	CONSTRAINT idx_index_aliquot UNIQUE ( name )
+ );
+ 
+ CREATE TABLE pm.shotgun_index_tech (
+	shotgun_index_tech_id bigserial  NOT NULL,
+	name                 varchar(100)  ,
```
Comment (wasade) [inline amgut/db/patches/0041.sql]: what is this one limited?

File: amgut/db/patches/0041.sql [hunk 2]
```diff
@@ -236,17 +244,17 @@ CREATE TABLE pm.water_lot (
     CONSTRAINT uq_water_lot_name UNIQUE ( name )
  );
 
-CREATE TYPE target_region AS ENUM ('16S', '18S', 'ITS');
-CREATE TYPE target_subfragment AS ENUM ('V4');
+CREATE TYPE pm.target_region AS ENUM ('16S', '18S', 'ITS');
```
Comment (wasade) [inline amgut/db/patches/0041.sql]: Should this differentiate ITS1, ITS2, overhanging (e.g., sequencing 3' of 18S into ITS1)? Perhaps as subfragment?

File: amgut/db/patches/0041.sql (hunk 1 above)
Comment (antgonza) [inline amgut/db/patches/0041.sql]: Honestly, cause that was the default in dbSchema and didn't catch it. However, this is just the name of the tech, which in general is like 20 chars ...

File: amgut/db/patches/0041.sql (hunk 2 above)
Comment (antgonza) [inline amgut/db/patches/0041.sql]: Good question, I just fixed the pm part of this but I can add more ... @josenavas

Comment (wasade) [inline amgut/db/patches/0041.sql]: Let's defer to later

**Pull request merged**